analyzer/evmtokens/evm_tokens.go

```diff
@@ -157,15 +157,23 @@ func (p *processor) ProcessItem(ctx context.Context, batch *storage.QueryBatch,
 		if err != nil {
 			return fmt.Errorf("downloading mutated token %s: %w", staleToken.Addr, err)
 		}
+		// We may be unable to download the totalSupply because
+		// of a deterministic error or because the contract may
+		// not provide the data.
 		if mutable != nil && mutable.TotalSupply != nil {
 			batch.Queue(queries.RuntimeEVMTokenDownloadedTotalSupplyUpdate,
 				p.runtime,
 				staleToken.Addr,
 				mutable.TotalSupply.String(),
-				staleToken.DownloadRound,
```

Comment:
note: was thinking of updating the `RuntimeEVMTokenDownloadedUpsert` query above, but I'm hesitating bc
- the query is already nicely atomic, and it'd be weird to insert a downloaded token without providing the round it was downloaded at immediately
- the subsequent update is idempotent.